src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java

```diff
@@ -72,7 +72,7 @@ public CommandManagerWindow(ICommandManager commandManager, DriverManager driver
         content.add(currentCommandField, c);
 
         this.driverManager = driverManager1;
-        driverManager.setCurrentDriver(new LineDriverAdapter(drawPanelController,new BasicLine(),"preview"));
+        //driverManager.setCurrentDriver(new LineDriverAdapter(drawPanelController,new BasicLine(),"preview"));
```

Comment:
Leaving such comments in the code is a bad practice. If the app works without this line, I think it should ok.